owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -286,9 +291,13 @@ abstract class DrawerActivity : ToolbarActivity() {
     }
 
     private fun openFeedback() {
-        val feedbackMail = getString(R.string.mail_feedback)
-        val feedback = "Android v" + BuildConfig.VERSION_NAME + " - " + getString(R.string.drawer_feedback)
-        sendEmail(email = feedbackMail, subject = feedback)
+        if (drawerViewModel.isFeedbackEnabled()) {
```

Comment:
i guess the `isFeedbackEnabled` condition refers the visibility, in case a branding customer does not want to use feedback. For that reason, the `openFeedbackDialog` should not be out of that condition. There are three mail parameters: 

`feedback_enabled`
`mail_recommend` (not totally sure here)
`mail_feedback`

condition to control should be `mail_feedback`. OTOH, i miss the removal of the current non-existing feedback email address from this PR.